.github/workflows/update-schema-dependency-version.yaml

```diff
@@ -1,37 +1,52 @@
-name: Update Version
+name: Update Schema Version
 
 on:
   workflow_call:
     inputs:
       major:
         description: 'Schema Major version'
+        type: string
         required: true
       minor:
         description: 'Schema Minor version'
+        type: string
         required: true
       patch:
         description: 'Schema Patch version'
+        type: string
+        required: true
+      downstream-repo-to-update:
+        description: 'The name of the downstream repository for which to update the schema version'
+        type: string
+        required: true
+      schema-property-name-in-repo:
+        description: 'The property name in the repos pom.xml for the schema version, e.g. rcsb.mojave.model.version'
+        type: string
         required: true
 
 
 env:
-  NEW_SCHEMA_VERSION: "${{ github.event.inputs.major }}.${{ github.event.inputs.minor }}.${{ github.event.inputs.patch }}"
+  NEW_SCHEMA_VERSION: "${{ inputs.major }}.${{ inputs.minor }}.${{ inputs.patch }}"
 
 jobs:
   update-version:
     runs-on:
       - self-hosted
       - java
     steps:
-      - uses: actions/checkout@v3
+      - name: Checking out ${{ inputs.downstream-repo-to-update }}
+        uses: actions/checkout@v3
+        with:
+          repository: rcsb/${{ inputs.downstream-repo-to-update }}
+          path: ${{ inputs.downstream-repo-to-update }}
       - name: Setup git author information
         run: |
           git config user.name RCSBSystem
           git config user.email it@rcsb.org
       - run: mvn --version
       - name: Set new schema dependency version
         run: |
-          mvn versions:set-property -Dproperty=rcsb.mojave.model.version -DnewVersion=$NEW_SCHEMA_VERSION -DgenerateBackupPoms=false
+          mvn versions:set-property -Dproperty=${{ inputs.schema-property-name-in-repo }} -DnewVersion=$NEW_SCHEMA_VERSION -DgenerateBackupPoms=false
```

Comment:
Isn't it always the same, is it?

Reply:
It's tricky: in rcsb-mojave-models it has one name (`rcsb-json-schema.version`), whilst in all others it has another name (`rcsb.mojave.model.version`)

Reply:
OK, I see, thanks, can we normalize that e.g. rename in mojave? What will be the impact?

Reply:
There is some logic for that (they are different names because they are actually different dependencies). So I'd keep it as it is. Passing the variable is not so bad, right? The python script works the same way

Reply:
Well, it introduces some noise though. If that logic is mojave specific, that also violates encapsulation. I would suggest to hide that property by adding a new one, named the same way as others, and inside mojave use the old one set to the value of a new one. This way we will localize that behavior inside mojave, and promote here a general idea of passing new schema version - which is our main goal here IMHO